.github/workflows/format.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       - name: Check project is formatted
         uses: jrouly/scalafmt-native-action@v1
         with:
-          version: '3.6.0'
+          version: '3.6.1'
```

Comment:
Just an idea, may be you could fetch the version from `.scalafmt.conf` and use it here as a property?

Reply:
Agreed, it is either that or making scala-steward aware of this github workflow. Just haven't had time for it recently.